core/core.go

```diff
@@ -194,6 +178,8 @@ type Config struct {
 
 	Hub   bool
 	Debug bool
+
+	TokenPowergateAdmin string
```

Comment:
This gets set from a flag and passed down into wherever uses the powergate admin client.